packages/i18n/src/test/index.js

```diff
@@ -34,13 +34,13 @@ const additionalLocaleData = {
 setLocaleData( localeData, 'test_domain' );
 
 describe( 'i18n', () => {
-	describe( 'dcnpgettext()', () => {
-		it( 'absorbs errors', () => {
-			const result = dcnpgettext( 'domain-without-data', undefined, 'Hello' );
+	it( 'absorbs errors', () => {
+		__( 'Hello', 'domain-without-data' );
```

Comment:
Should it call `expect( console ).toHaveErrored();` after each function call instead?

Reply:
>Should it call `expect( console ).toHaveErrored();` after each function call instead?

Good call. Updated in rebased 1f0c639.